Syntaxes/Dockerfile.sublime-syntax

```diff
@@ -1,114 +1,57 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
+# Syntax: https://www.sublimetext.com/docs/syntax.html
 name: Dockerfile
-file_extensions:
-  - Dockerfile
-  - dockerfile
-first_line_match: ^\s*(?i:(from(?!\s+\S+\s+import)|arg))\s+
-scope: source.dockerfile
+comment: "Dockerfile Syntax: v1.5.3"
+file_extensions: [Dockerfile]
+first_line_match: '^\s*(?i:(from(?!\s+\S+\s+import)|arg))\s+'
+uuid: 1c940d5b-ff4d-4d03-854b-701bffc44912
+scope: source.Dockerfile
+extends: Packages/Dockerfile/Dockerfile.sublime-syntax
 
 variables:
-  onbuild_directive: (?i:(onbuild)\s+)?
-  onbuild_commands_directive:
-    "{{onbuild_directive}}(?i:add|arg|env|expose|healthcheck|label|run|shell|stopsignal|user|volume|workdir)"
-  nononbuild_commands_directive: (?i:maintainer)
-  runtime_directive: "{{onbuild_directive}}(?i:cmd|entrypoint)"
-  from_directive: (?i:(from))\s+[^\s:@]+(?:[:@](\S+))?(?:\s+(?i:(as))\s+(\S+))?
-  copy_directive: ({{onbuild_directive}}(?i:copy))(?:\s+--from=(\S+))?
+  control_keywords: '^(FROM|CMD|COPY|WORKDIR|RUN|ENV|ADD|LABEL|STOPSIGNAL|USER|VOLUME|ONBUILD|EXPOSE|MAINTAINER|ARG|ENTRYPOINT|SHELL)'
 
 contexts:
   main:
-    - include: comments
-    - match: ^(?i:arg)\s
-      scope: keyword.control.dockerfile
-    - include: from
-
-  from:
-    - match: ^{{from_directive}}
-      captures:
-        1: keyword.control.dockerfile
-        2: entity.name.enum.tag-digest
-        3: keyword.control.dockerfile
-        4: variable.stage-name
-      push: body
-
-  body:
-    - include: comments
-    - include: directives
-    - include: invalid
-    - include: from
-
-  directives:
-    - match: ^\s*{{onbuild_commands_directive}}\s
-      captures:
-        0: keyword.control.dockerfile
-        1: keyword.other.special-method.dockerfile
-      push: args
-    - match: ^\s*{{nononbuild_commands_directive}}\s
-      scope: keyword.control.dockerfile
-      push: args
-    - match: ^\s*{{copy_directive}}\s
-      captures:
-        1: keyword.control.dockerfile
-        2: keyword.other.special-method.dockerfile
-        3: variable.stage-name
-      push: args
-    - match: ^\s*{{runtime_directive}}\s
-      captures:
-        0: keyword.operator.dockerfile
-        1: keyword.other.special-method.dockerfile
-      push: args
-
-  escaped-char:
-    - match: \\.
-      scope: constant.character.escaped.dockerfile
-
-  args:
-    - include: comments
-    - include: escaped-char
-    - match: ^\s*$
-    - match: \\\s+$
-    - match: \n
-      pop: true
+    - include: keyword
+    - include: flag
+    - include: comment-single-line
+    - include: operator-word
+    - include: const
+    - include: string-double
+
+  keyword:
+    - match: '^(FROM|CMD|COPY|WORKDIR|RUN|ENV|ADD|LABEL|STOPSIGNAL|USER|VOLUME|ONBUILD|EXPOSE|MAINTAINER|ARG|ENTRYPOINT|SHELL)'
+      scope: keyword.control.Dockerfile
+
+  operator-word:
+    - match: \b(apt\-get|get|install|clean|rm\s*\-rf|pip3|pip|python3|python|echo|exit)\b
```

Comment:
I think this is too specific. This PR would be great to be merged with https://github.com/asbjornenge/Docker.tmbundle/pull/24